app/components/SchoolTimelineInfo.js

```diff
@@ -39,11 +39,14 @@ function processRow(details) {
       boxContent: 'sijainti'
     },
     {
-      // We are not getting the archive address from the API yet.
       key: 'school-archive',
       className: 'details-archive',
-      name: getAddressArrayFromBuilding(details.building.building),
-      boxContent: 'arkisto'
+      name: details.archive.location,
+      boxContent: [
+        details.archive.begin_year,
+        <i className='fa fa-lg fa-long-arrow-right'/>,
+        details.archive.end_year
+      ]
```

Comment:
This information is in the timeline so maybe the boxContent could be perhaps Arkistot, like with Rehtori and Sijainti. 

Following this same logic perhaps the name of the school should say name as well ? :confused: